hammer/reporting-remediation/reporting/create_elasticsearch_unencrypted_issue_tickets.py

```diff
@@ -0,0 +1,163 @@
+                                ticket_id=issue.jira_details.ticket,
+                                label=IssueStatus.Whitelisted.value
+                            )
+                        jira.close_issue(
+                            ticket_id=issue.jira_details.ticket,
+                            comment=comment
+                        )
+                        slack.report_issue(
+                            msg=f"{comment}"
+                                f"{' (' + jira.ticket_url(issue.jira_details.ticket) + ')' if issue.jira_details.ticket else ''}",
+                            owner=owner,
+                            account_id=account_id,
+                            bu=bu, product=product,
+                        )
+                        IssueOperations.set_status_closed(ddb_table, issue)
+                    # issue.status != IssueStatus.Closed (should be IssueStatus.Open)
+                    elif issue.timestamps.updated > issue.timestamps.reported:
+                        logging.error(f"TODO: update jira ticket with new data: {table_name}, {account_id}, {domain_name}")
+                        slack.report_issue(
+                            msg=f"Elasticsearch unencrypted domain '{domain_name}' issue is changed "
+                                f"in '{account_name} / {account_id}' account, '{region}' region"
+                                f"{' (' + jira.ticket_url(issue.jira_details.ticket) + ')' if issue.jira_details.ticket else ''}",
+                            owner=owner,
+                            account_id=account_id,
+                            bu=bu, product=product,
+                        )
+                        IssueOperations.set_status_updated(ddb_table, issue)
+                    else:
+                        logging.debug(f"No changes for '{domain_name}'")
+                # issue has not been reported yet
+                else:
+                    logging.debug(f"Reporting Elasticsearch unencrypted domain '{domain_name}' issue")
+
+                    owner = tags.get("owner", None)
+                    bu = tags.get("bu", None)
+                    product = tags.get("product", None)
+
+                    issue_description = (
+                        f"Elasticsearch domain needs to be encrypted.\n\n"
+                        f"*Risk*: High\n\n"
+                        f"*Account Name*: {account_name}\n"
+                        f"*Account ID*: {account_id}\n"
+                        f"*Region*: {region}\n"
+                        f"*Domain ID*: {domain_name}\n"
+                    )
+
+                    issue_description += JiraOperations.build_tags_table(tags)
+
+                    issue_description += (
+                        f"*Recommendation*: Encrypt Elasticsearch domain. To enable encryption follow below steps: \n"
+                        f"1. Choose to create new domain. \n"
+                        f"2. Enable node-node encryption or encryption at rest options.\n"
```

Comment:
Again, why we should enable either encryption at rest or encryption in transit and not both?

Reply:
Ok. Updated issue remediation description to encrypt for both (at rest and transit)